paasta_tools/utils.py

```diff
@@ -2473,11 +2463,6 @@ class DeploymentsJsonV2Dict(TypedDict):
     controls: Dict[BranchName, _DeploymentsJsonV2ControlsDict]
 
 
-class DeploymentsJsonDict(TypedDict):
```

Comment:
@EvanKrall this was probably created with the intention of using mypy with it somehow? But It doesn't seem to be actually used anywhere. OK to delete?

Reply:
for context, it was added in this commit but never referenced: 3b7b0705e7a95e98f759b6b4be0ca9cf45278732

man I should really make a tool which spits out these "add in / last referenced in" blurbs :laughing: